### PR TITLE
[IMP] medical_diagnostic_report: Allow to automatically increase height

### PR DESCRIPTION
**Pull request by etobella**

### CODE REVIEW

**Comment by etobella** [comment]
@albariera Can you review it please?

File: medical_diagnostic_report/models/medical_diagnostic_report_template.py [hunk 1]
```diff
@@ -7,15 +7,16 @@
 class MedicalDiagnosticReportTemplate(models.Model):
     _name = "medical.diagnostic.report.template"
     _inherit = ["medical.report.abstract"]
-    _description = "Diagnostic Report Template"
+    _description = "Diagnostic Report Tmany2oneemplate"
```
Comment (LoisRForgeFlow) [inline medical_diagnostic_report/models/medical_diagnostic_report_template.py]: typo here?

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Comment by albariera** [comment]
Field name in medical_diagnostic_report_template shouldn't it be translate=True too? In case we do not have a title, the field name in the report will be set as the name of the template, so I think it should also be translated. Please correct me if I am wrong!

**Comment by etobella** [comment]
I know it could be, but I wanted to avoid too many translation fields :laughing:

**Comment by albariera** [comment]
Ok! So LGTM :)

**Review by albariera — APPROVED** [review]

**Pull request merged**